### ISSUE TITLE
nwely joining unit unable to hear LR talking unit

### ISSUE THREAD

**Issue by vabene1111** [michail-nikolaev/task-force-arma-3-radio#1470]
**Arma 3 Version:** `1.92.145639`
**CBA Version:** `3.10.1.190316` 
**Task Force Radio Version:** `1.-1.0.318` 

**Mods:**
```
- @CBA_A3
- @task_force_radio
```
**Description:**
When a player holds down the radio button (tested with LR radio) and another player joins the server while the other one keeps the button presses, the newly joined player wont be able to hear the unit that is holding down the radio button until it is released

**Steps to reproduce:**
- Player A joins the server
- Player A holds down LR Radio button 
- Player B joins the server
- Player B wont be able to hear player A until player A releases the button

**Where did the issue occur?**
- Tested on a dedicated server in an empty VR mission with just two units and standard (except for channel name) tfar settings

**RPT log file:**
- Server https://gist.github.com/vabene1111/0b48c62adbe776a356879067988ac24d
- Player A https://gist.github.com/vabene1111/744799dfeafcf759c964df4cb2b30d55
- Player B https://pastebin.com/4YaijG4y

I hope i haven't missed some other issue reporting the same problem, if so, i am sry but maybe this still helps :)

**Comment by Fusselwurm** [comment]
(how did you notice that? are you streaming audio books via TFAR or something?)

**Comment by dedmen** [comment]
Correct. TFAR sends out one message to all users in channel that "hey! I just started transmitting on this radio with this range"

I guess client could request a update from everyone about their current status. I already wanted to implement that for voice volume anyway.

**Comment by vabene1111** [comment]
Yes this is usually no big problem, just something we noticed the other day and wanted to let you know :)

**Comment by vabene1111** [comment]
@Fusselwurm actually we developed a very hacky script that lets people talk continously together (like on a phone). I think there was, at some point, the plan to integrate a phone directly in tfar since some effects already exist for it but i guess (as so often with dev stuff) something more important came up 😂 

i first thought it was a problem with my script but since it could be reproduced with just vanialla TFAR and CBA i thought i better report it :)
